src/Command/Release2.php

```diff
@@ -283,10 +290,22 @@ protected function release()
         $this->stdio->outln('Getting the tagged release.');
         $this->shell('git pull');
 
-        $this->followupEmail();
+        // $this->followupEmail();
+        $this->followupEmailToQueue();
```

Comment:
may be you can change 

```php
if (! $this->followupEmailToQueue()) {
    $this->followupEmail();
}
``` 

to make sure the mails are send if the size of the message is more than 64 Kb.